src/Components/CriticalCareRecording/CriticalCare__API.tsx

```diff
@@ -24,3 +26,17 @@ export const updateDailyRound = (
     id,
   });
 };
+
+export const getAsset = (
+  consultationId: string,
+  setAsset: React.Dispatch<React.SetStateAction<number>>
+) => {
+  request(routes.listConsultationBeds, {
+    query: { consultation: consultationId },
+  }).then((result) => {
```

Comment:
Pass a `limit` of 1 if only the latest (first) consultation bed record is only required.